src/components/form/TwitterForm.js

```diff
@@ -88,120 +89,151 @@ const TwitterForm = () => {
-          <Option value="Georgia">Georgia</Option>
-          <Option value="Hawaii">Hawaii</Option>
-          <Option value="Idaho">Idaho</Option>
-          <Option value="Illinois">Illinois</Option>
-          <Option value="Indiana">Indiana</Option>
-          <Option value="Iowa">Iowa</Option>
-          <Option value="Kansas">Kansas</Option>
-          <Option value="Kentucky">Kentucky</Option>
-          <Option value="Louisiana">Louisiana</Option>
-          <Option value="Maine">Maine</Option>
-          <Option value="Maryland">Maryland</Option>
-          <Option value="Massachusetts">Massachusetts</Option>
-          <Option value="Michigan">Michigan</Option>
-          <Option value="Minnesota">Minnesota</Option>
-          <Option value="Mississippi">Mississippi</Option>
-          <Option value="Missouri">Missouri</Option>
-          <Option value="Montana">Montana</Option>
-          <Option value="Nebraska">Nebraska</Option>
-          <Option value="Nevada">Nevada</Option>
-          <Option value="New Hampshire">New Hampshire</Option>
-          <Option value="New Jersey">New Jersey</Option>
-          <Option value="New Mexico">New Mexico</Option>
-          <Option value="New York">New York</Option>
-          <Option value="North Carolina">North Carolina</Option>
-          <Option value="North Dakota">North Dakota</Option>
-          <Option value="Ohio">Ohio</Option>
-          <Option value="Oklahoma">Oklahoma</Option>
-          <Option value="Oregon">Oregon</Option>
-          <Option value="Pennsylvania">Pennsylvania</Option>
-          <Option value="Rhode Island">Rhode Island</Option>
-          <Option value="South Carolina">South Carolina</Option>
-          <Option value="South Dakota">South Dakota</Option>
-          <Option value="Tennessee">Tennessee</Option>
-          <Option value="Texas">Texas</Option>
-          <Option value="Utah">Utah</Option>
-          <Option value="Vermont">Vermont</Option>
-          <Option value="Virginia">Virginia</Option>
-          <Option value="Washington">Washington</Option>
-          <Option value="West Virginia">West Virginia</Option>
-          <Option value="Wisconsin">Wisconsin</Option>
-          <Option value="Wyoming">Wyoming</Option>
-        </Select>
-
-        <label htmlFor="incident_date" className="incident_date">
-          Date
-        </label>
-        {/* Potential project for upcoming cohort, we ran out of time to figure out how defaultValue works for DatePicker from antd
+        <br></br>
+        <label htmlFor="incident_date" className="form-labels">
+          <b>Date</b>
+          <br></br>
+          {/* Potential project for upcoming cohort, we ran out of time to figure out how defaultValue works for DatePicker from antd
         Somehow, we have to be able to convert the "zulu timestamp" to a regular date statement YYYY-MM-DD. Since we were unable
         to do this, we've decided to keep the date blank for our submission. */}
```

Comment:
Love the documentation you have here for future cohorts. Surprised you guys weren't able ti convert the Zulu time delta.